scene/resources/resource_format_text.cpp

```diff
@@ -1666,6 +1716,27 @@ String ResourceFormatLoaderText::get_resource_type(const String &p_path) const {
 	return ClassDB::get_compatibility_remapped_class(r);
 }
 
+String ResourceFormatLoaderText::get_resource_script_class(const String &p_path) const {
+	String ext = p_path.get_extension().to_lower();
+	if (ext == "tscn") {
+		return "";
+	} else if (ext != "tres") {
+		return String();
+	}
```

Comment:
Same here. `tscn` check is redundant.
Although I assume you just copied it from somewhere...